blockbase/inc/disable-site-editor.php

```diff
@@ -76,16 +80,17 @@ function_exists( 'gutenberg_use_widgets_block_editor') &&
 /**
  * Removes the Site Editor link from the admin.
  */
-function remove_site_editor_admin_link() {
+function blockbase_remove_site_editor_admin_link() {
 	global $menu;
+
 	// Remove Site Editor.
 	foreach ( $menu as $index => $menu_item ) {
-		if ( ! empty( $menu_item[5] ) && false !== strpos( $menu_item[5], 'toplevel_page_gutenberg-edit-site' ) ) {
+		if ( ! empty( $menu_item[ 2 ] ) && in_array( $menu_item[ 2 ], array( 'gutenberg-edit-site', 'site-editor' ) ) ) {
```

Comment:
On dotorg it's called `gutenberg-edit-site` on dotcom it's called `site-editor`.

Reply:
How do we know to check the index `2` for the presence of the Site Editor?